src/components/portfolio/DetailsHeader.test.tsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
+import { ThemeProvider } from '@material-ui/core/styles';
 import DetailsHeader from './DetailsHeader';
 import { MockPositions } from '../../portfolio/APIMocks';
+import { theme } from '../App';
 
 const testName = 'My awesome portfolio';
 const mockBack = jest.fn();
 
 test('Details header renders correctly', async () => {
   render(
-    <DetailsHeader
-      back={mockBack}
-      name={testName}
-      positions={MockPositions}
-      editPositions={jest.fn()}
-    />
+    <ThemeProvider theme={theme}>
```

Comment:
Why is the theme provider needed here?

Reply:
I changed hardcoded color codes to `lightBlue` from the theme. Without the `ThemeProvider` wrapper, the tests fail because the theme color does not exist.